### PR TITLE
Exclude slow detectors from stopflow scans

### LINKED ISSUE #63: Include OAV in stopflow experiment
https://github.com/DiamondLightSource/i22-bluesky/issues/35#issuecomment-2165319195

### PR DESCRIPTION
**Pull request by callumforrester**
Fixes #63 and #65

### CODE REVIEW

File: src/i22_bluesky/plans/stopflow.py [hunk 1]
```diff
@@ -371,3 +377,29 @@ def stopflow_seq_table(
...
+def raise_for_minimum_exposure_times(
+    exposure: float,
+    detectors: list[StandardDetector],
+) -> None:
+    minimum_exposure_times = {
+        "saxs": 1.0 / 250.0,
+        "waxs": 1.0 / 250.0,
+        "oav": 1.0 / 22.0,
+        "i0": 1.0 / 2e4,
+        "it": 1.0 / 2e4,
```
Comment (DiamondJoseph) [inline src/i22_bluesky/plans/stopflow.py:391]: Where have these numbers come from? Can we get it from a signal?

**Review by DiamondJoseph — APPROVED** [review]

File: src/i22_bluesky/plans/stopflow.py (hunk 1 above)
Comment (callumforrester) [reply src/i22_bluesky/plans/stopflow.py:391]: No, per @coretl they are _very_ difficult to work out dynamically. Hard-coding them in a lookup table will do for now, that may actually be the solution forever...

Comment (coretl) [reply src/i22_bluesky/plans/stopflow.py:391]: It's all down to how much effort we want to put into this. The max framerate, min exposure time, deadtime etc. may depend on multiple parameters, and require many hours of scouring manuals and manual experimentation to discover. Is it worth it to discover them in a general way or should we put them in the beamline specific plan?

Comment (callumforrester) [reply src/i22_bluesky/plans/stopflow.py:391]: I think the answer here is to allow as much or as little complexity as one is willing to put in. For example, on a beamline where speed is less important they may just put in a worst-case-scenario, slowest possible max speed. For a performance-focused beamline, they do all the manual experimentation and enter several values/write logic to work it out depending on various parameters.

**Pull request merged**